integration/loader/main.go

```diff
@@ -28,6 +28,7 @@ type LoadCase struct {
 func main() {
 
 	sleep := flag.Duration("sleep", 0, "duration to sleep before benchmarking starts")
+	loadTypeFlag := flag.Int("load-type", 0, "type of loads (token transfer, set keys, ...)s")
```

Comment:
Could we make this a string flag instead? `--load-type=token-transfer` is a lot more readable than `--load-type=0`